### DISCUSSION TITLE
2023-11-13

### DISCUSSION THREAD

**Discussion by oauch** [10-rolling/rollingPolling#84] [Daily_Meeting]
### ✅ 어제 해낸 일

- [x] Post 페이지에 카드 리스트 출력
- [x] arrowdown 망령 퇴치
- [x] DropDown 스타일 수정
- [x] PostList 카드에 해당하는 폰트 적용
- [x] Editor 컴포넌트 입력값 버그 해결
- [x] list 페이지 거의 완성

### 📝 회의내용
- 오늘의 TMI
윤혁 : 더워서 잠 못잠 
슬희님 : PT 쌤 변경, 19:30 새로운 선생님
수님 : 너무 추워서 감기 걸림... 약먹고 옷꽁으로 잠

- 각자 주말동안 한거 발표
윤혁 : DropDown 스타일 수정, Editor 컴포넌트 입력값 버그 해결, PostList 카드에 해당하는 폰트 적용
슬희님 : list 페이지
수님 : 열심히 놀았음
수영님 : 화이팅

- 다음 할일 역할분담
수영님 : 무한 스크롤....
수님 : 카드 삭제하기, 카드 **지인 스타일**
슬희님 : 배포, list 페이지 수정
윤혁 : 반응형 페이지 + 스타일 수정

### 💪🏻 오늘 할일

- 개발하기
- **운동하기**
- 개발하기

### 🤔 특이사항

이번주 프로젝트 마지막이네요
다들 화이팅 입니다 🙂

**Comment by hanseulhee** [comment]
화이팅 ~~~~~~~

**Comment by ngsy** [comment]
화이팅!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by miniposi** [comment]
ez~
